guide/resources/design-research.md

```diff
@@ -97,6 +97,13 @@ A survey and follow-up interview of 990 bitcoin users to determine bitcoin manag
 
 ## Independent research
 
+#### [Bitcoin: Knowledge and Perceptions]
+(https://block.xyz/2022/btc-report.pdf)
```

Comment:
The link is not rendering correctly because there is a line break between the `]` and the `/`.

Reply:
Thanks for the tips and for correcting this.